gptel-transient.el

```diff
@@ -1128,6 +1129,16 @@ supports.  See `gptel-track-media' for more information."
 (declare-function gptel-context--at-point "gptel-context")
 (declare-function gptel-add "gptel-context")
 
+(transient-define-suffix gptel--infix-context-add-current-kill (&optional arg)
+  "Add current kill to gptel's context."
+  :transient 'transient--do-stay
+  :key "C-y"
+  :description
```

Comment:
Please make this an expert-command, as I'm trying to keep first-time users from running away after looking at the menu.

You can add
```emacs-lisp
:if (lambda () gptel-expert-commands)
```
to the transient-suffix specification.

Reply:
This is a really good point.